README.md

```diff
@@ -24,7 +24,7 @@
 $ git clone https://github.com/mist64/xhyve
 $ cd xhyve
 $ make
-$ cp build/xhyve /usr/local/bin/ # You may need sudo.
+$ cp build/xhyve /usr/local/bin/    # You may be asked for your sudo password
```

Comment:
Sorry for this again.
It is OK with `You may need sudo` in this case, because you may have to put `sudo` to copt it into the `/usr/local/bin/`.